test/performance/odp_l2fwd.c

```diff
@@ -791,6 +792,18 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx,
 
 	odp_pktio_config(pktio, &config);
 
+	if (pktio_capa.set_op.op.promisc_mode) {
+		bool promisc_mode = gbl_args->appl.promisc_mode;
+
+		/* Enable/Disable promisc mode */
+		if (odp_pktio_promisc_mode_set(pktio, promisc_mode)) {
+			ODPH_ERR("Error: promisc mode set failed %s\n", dev);
+			return -1;
+		}
+		printf("Promisc mode %s\n",
+		       promisc_mode ? "enabled" : "disabled");
```

Comment:
This print would fit better inside `print_info()` function.

Reply:
Ok.